zppy/global_time_series.py

```diff
@@ -74,18 +117,30 @@ def global_time_series(config, scriptDir, existing_bundles, job_ids_file):
             # List of dependencies
             dependencies = []
             # Add Time Series dependencies
-            # Iterate from year1 to year2 incrementing by the number of years per time series file.
-            for yr in range(c["year1"], c["year2"], c["ts_num_years"]):
-                start_yr = yr
-                end_yr = yr + c["ts_num_years"] - 1
-                dependencies.append(
-                    os.path.join(
-                        scriptDir,
-                        "ts_%s_%04d-%04d-%04d.status"
-                        % ("atm_monthly_glb", start_yr, end_yr, c["ts_num_years"]),
+            if c["use_atm"]:
```

Comment:
Dependency handling had to be updated to properly match what components need plotting.